boards/imix/src/udp_lowpan_test.rs

```diff
@@ -412,6 +463,229 @@ impl<'a, A: time::Alarm<'a>> LowpanTest<'a, A> {
         self.mock_udp1.bind(16123);
         self.mock_udp2.bind(16124);
     }
+
+    // Test network capability enforcement for addrs
+    // TODO: include a positive and negative test for each (as apporpriate)
```

Comment:
has this TODO been addressed?

Reply:
Done! Yes -- the tests have different combinations of UDP/IP capabilities being valid/invalid (in terms of permissiveness)